daffodil-lib/src/main/scala/org/apache/daffodil/lib/xml/DaffodilXMLLoader.scala

```diff
@@ -251,26 +240,32 @@ class DFDLCatalogResolver private ()
         None
       }
       case (null, sysId) => {
-        // We did not get a catalog resolution of the nsURI, nor
-        // a straight file resolution of the systemId, so we
-        // use the systemId (which comes from the schemaLocation attribute)
-        // and the classpath.
-        val baseURI = if (baseURIString == null) None else Some(new URI(baseURIString))
-        val optURI = Misc.getResourceRelativeOption(sysId, baseURI)
+        // We did not get a catalog resolution of the nsURI. We now look for the systemID (which
+        // comes from the schemaLocation attribute) on classpath or as a file. Note that we
+        // ignore if boolean
```

Comment:
ignore if boolean..... what

Reply:
This "note" can be removed. It was saying that the boolean part of the tuple returned by resolveSchemaLocation can be ignored since we look at it elsewhere, but I move it as a comment a couple lines below and I guess I messed up removing this sentence. I'll remove this.